extensions-core/namespace-lookup/src/main/java/io/druid/query/extraction/namespace/URIExtractionNamespace.java

```diff
@@ -64,40 +64,58 @@
   @JsonProperty
   private final URI uri;
   @JsonProperty
+  private final URI uriPrefix;
+  @JsonProperty
   private final FlatDataParser namespaceParseSpec;
   @JsonProperty
-  private final Period pollPeriod;
+  private final String fileRegex;
   @JsonProperty
-  private final String versionRegex;
+  private final Period pollPeriod;
 
   @JsonCreator
   public URIExtractionNamespace(
```

Comment:
I'd rather leave it like it is in this PR. in the bigger migration PR I can break more things.